frameworks/digital-outcomes-and-specialists-5/questions/brief-responses/availability.yml

```diff
@@ -10,13 +10,10 @@ question_advice: >
   {%- else %}
   participants: {{ brief.researchDates }}
   {%- endif %}
-type: text
-smaller: true
+type: date
 validations:
-  -
-    name: answer_required
-    message: 'You need to answer this question.'
-  -
-    name: under_character_limit
-    message: "Your start date must be no more than 100 characters."
+  - name: answer_required
+    message: Enter a start date
+  - name: invalid_format
+    message: Start date must be a real date
```

Comment:
Do we want these to be consistent with existing questions with type `date` - e.g. `frameworks/digital-outcomes-and-specialists-5/questions/briefs/awardedContractStartDate.yml`?

Reply:
Yeah good idea

Reply:
Hmm, actually looking at it closer I think the other questions need to have their error messages updated 🤔 They are currently not aligned with Design System guidance

This should happen some time before DOS5 is opened; if we don't have a ticket for it already I will create one now.